### DISCUSSION TITLE
Floating Side Panels

### DISCUSSION THREAD

**Discussion by BatAffl3ck** [Floorp-Projects/Floorp#458] [Feature Request]
### Is your feature request related to a problem?
Side Panels right now work as a split screen feature and move the current tab and its content aside while open.

### Describe the solution you'd like
It would be great if side panels do not move the current tab and its content but instead, they float over the tab content kind of like in vivaldi/edge/opera, etc. This way side panel becomes more productive. 

### Describe alternatives you've considered
No response

### Additional context
No response

**Comment by Ryosuke-Asano** [comment]
We have thinking about this feature several times.

[But it is difficult...](url)

**Reply by bakinazik** [reply]
is any progress? the url is missing so i can't check what's going on inside..

**Reply by KaKi87** [reply]
Hi,

A year and seven months later, any news on this ?

Thanks

**Comment by breadbeforerice** [comment]
The left side is possible here. I hope someone will show up who can do the right side.
https://old.reddit.com/r/Floorp/comments/18rq83q/make_the_operalike_sidebar_float/

**Comment by breadbeforerice** [comment]
The one for the right side has also been completed.(Assumes a screen resolution of 1920px horizontally)
#sidebar2-box{
height: 100% !important;
  z-index: 1;
  height: 100%;
  position: absolute;
  right: 42px;
}

html[sizemode=fullscreen] #sidebar2-box{
height: 100% !important;
position: absolute !important;
z-index:1 !important;
padding-left: 0px !important;
left:1521px;
border: none !important;
}

#sidebar-splitter2{
display: none !important;
}

**Reply by breadbeforerice** [reply]
left:1521px; corresponds to a web page width of 400px, so change it if you change the width. So if you mix different values, it will look wrong.
Edit: My mistake. The tab actually moves slowly and is the width you specified.

**Comment by zenwav** [comment]
I'd like to second this proposal. I would prefer the sidebar to display over the current webpage instead of shifting the webpage to the left.

**Reply by ioc2e3** [reply]
> ... . I would prefer the sidebar to display over the current webpage instead of shifting the webpage to the left.

Agree ...

.
